consumers/google.go

```diff
@@ -65,10 +56,21 @@ func NewGoogleDNS() (Consumer, error) {
 		return nil, fmt.Errorf("Error creating DNS service: %v", err)
 	}
 
+	resp, err := client.ManagedZones.List(params.project).Do()
```

Comment:
shouldn't this be part of the Process/Sync flow ? Querying it once for static client is probably okay most of the time, but newly created zones won't be picked up by Mate.